_layouts/base.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <!-- favicon -->
+     <!-- <link rel="icon" type="image/png" href="{{ site.baseurl }}/images/trans/6.png" /> -->
+    <link rel="icon" type="image/png" href="{{ site.baseurl }}/images/color/favicon180.png" />
+
+    {% if page.title == nil %}
+        <meta name="title" content="Forseti Security">
+        <meta name="og:title" content="Forseti Security">
+    {% else %}
+        <meta name="title" content="{{ page.title }}">
+        <meta name="og:title" content="{{ page.title }}">
+    {% endif %}
+
+    {% if page.overview == nil %}
+        <meta name="description" content="{{ site.description }}">
+        <meta name="og:description" content="{{ site.description }}">
+    {% else %}
+        <meta name="description" content="{{ page.overview }}">
+        <meta name="og:description" content="{{ page.overview }}">
+    {% endif %}
+
+    {% if page.title == nil %}
+        <title>Forseti Security</title>
+    {% else %}
+        <title>Forseti Security / {{ page.title }}</title>
+    {% endif %}
+
+    <!-- TODO: Add Google Analytics -->
```

Comment:
Can we confirm this? Because Google Analytics was being added by the _jekyll-theme-leap-day_ previously, which has been removed. If Travis CI does indeed include Google Analytics, then I will not add.